tests/cpp_random_unit.cpp

```diff
@@ -482,7 +482,7 @@ DEFINE_TEST(random_doublecheck_test_64) {
 int main() {
     uint64_t seed = time(nullptr);
     srand(seed);
-    printf("Seed: %lu\n", seed);
+    printf("Seed: %llu\n", seed);
```

Comment:
It's unfortunate but this is not portable.  On some systems, `int` is 32bits and `long` and `long long` are 64bits.  On those systems, `uint64_t` often is an alias for `long`, not `long long`, and `%llu` fails if you have format string checking enabled.  See: https://godbolt.org/z/6zr1ndq86

 The new C++ `std::fmt` facility fixes all this and is much nicer than printf, but unfortunately it's not available in older versions of C++.  You will probably have to add a cast to `unsigned long long` if you want to avoid the format string warnings on all systems.

Reply:
Done. See my last commit.